diff --git a/node_modules/@aws-amplify/pubsub/dist/esm/Providers/MqttOverWS.mjs b/node_modules/@aws-amplify/pubsub/dist/esm/Providers/MqttOverWS.mjs
index 0e2ac29..da672b8 100644
--- a/node_modules/@aws-amplify/pubsub/dist/esm/Providers/MqttOverWS.mjs
+++ b/node_modules/@aws-amplify/pubsub/dist/esm/Providers/MqttOverWS.mjs
@@ -1,4 +1,4 @@
-import * as Paho from '../vendor/paho-mqtt.js';
+// import * as Paho from '../vendor/paho-mqtt.js';
 import { Observable } from 'rxjs';
 import { AbstractPubSub } from './PubSub.mjs';
 import { ConnectionState } from '../types/PubSub.mjs';
@@ -8,6 +8,10 @@ import { ConnectionStateMonitor, CONNECTION_CHANGE } from '../utils/ConnectionSt
 import { ReconnectionMonitor, ReconnectEvent } from '../utils/ReconnectionMonitor.mjs';
 import { CONNECTION_STATE_CHANGE } from './constants.mjs';
 
+import { createRequire } from 'module';
+const _require = createRequire(import.meta.url);
+const Paho = _require('../vendor/paho-mqtt.js');
+
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 // @ts-ignore
